src/legacy/ui/ui_render/ui_render_mixin.js

```diff
@@ -55,7 +55,8 @@ import { getApmConfig } from '../apm';
  */
 export function uiRenderMixin(osdServer, server, config) {
   const translationsCache = { translations: null, hash: null };
-  const defaultLocale = i18n.getLocale() || 'en'; // Fallback to 'en' if no default locale is set
+  // `i18n.getLocale` will always return a value; the 'en' is just to accommodate tests
+  const defaultLocale = i18n.getLocale() || 'en';
```

Comment:
will this always be comparable to a normalized locale? (given toLowerCase was dropped)

Reply:
Everyone now goes through `normalizeLocale`, as opposed to `.toLowerCase()` which is what the older normalize was doing.